components/ScrollToTop.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
add `"use client"` here 

```suggestion
"use client"
import React, { useEffect, useState } from 'react';
```